hw_04/task_03.js

```diff
@@ -0,0 +1,24 @@
+// Преобразовать Task 2* таким образом, чтобы age принимался через prompt в привязанной вёрстке, а результат выводился в alert
+
+
+const age_1 = 18
+const age_2 = 60
+
+const checkAge = prompt('Please, enter your age', []);
+
+if (isNaN(checkAge) || checkAge < 0 || checkAge > 120 || checkAge == " " || checkAge == null) {
```

Comment:
Same as for previous task. But here prompt returns only string values. So no need to check for null. Only NaN, empty string and string with spaces